### ISSUE TITLE
Kendo Vue Wrapper: Invalid template xxxx Generated code xxxx

### ISSUE THREAD

**Issue by novalagung** [telerik/kendo-ui-core#4944]
### Bug report

`yarn serve` generates no error, but `yarn build` does.

### Reproduction of the problem

What I did was quite straight forward, I import `@progress/kendo-*` components, then register it to `Vue.use()`. So I'm not even use the component yet, just imported it but already got the errors. below is my vue `main.js`:

```js
// ...
import '@progress/kendo-ui'
import '@progress/kendo-theme-default/dist/all.css'
import { GridInstaller, DataSourceInstaller } from '@progress/kendo-all-vue-wrapper'

Vue.use(GridInstaller)
Vue.use(DataSourceInstaller)

new Vue({
  router,
  render: h => h(App)
}).$mount('#app')
```

When I run the app using `yarn serve`, everything was going smoothly. But if I consume the distributable files generated from `yarn build` command, then underneath error messages showing up in the dev tools console.

```
Uncaught Error: Invalid template:'<div class="k-group-indicator" data-#=data.ns#field="${data.field}" data-#=data.ns#title="${data.title || ""}" data-#=data.ns#dir="${data.dir || "asc"}"><a href="\#" class="k-link"><span class="k-icon k-i-sort-${(data.dir || "asc") == "asc" ? "asc-sm" : "desc-sm"}" title="(sorted ${(data.dir || "asc") == "asc" ? "ascending": "descending"})"></span>${data.title ? data.title: data.field}</a><a class="k-button k-button-icon k-bare"><span class="k-icon k-i-close"></span></a></div>' Generated code:'var $kendoOutput, $kendoHtmlEncode = kendo.htmlEncode;$kendoOutput='<div class="k-group-indicator" data-'+(data.ns)+'field="'+($kendoHtmlEncode(data.field))+'" data-'+(data.ns)+'title="'+($kendoHtmlEncode(data.title || ""))+'" data-'+(data.ns)+'dir="'+($kendoHtmlEncode(data.dir || "asc"))+'"><a href="#" class="k-link"><span class="k-icon k-i-sort-'+($kendoHtmlEncode((data.dir || "asc") == "asc" ? "asc-sm" : "desc-sm"))+'" title="(sorted '+($kendoHtmlEncode((data.dir || "asc") == "asc" ? "ascending": "descending"))+')"></span>'+($kendoHtmlEncode(data.title ? data.title: data.field))+'</a><a class="k-button k-button-icon k-bare"><span class="k-icon k-i-close"></span></a></div>';return $kendoOutput;'
    at Object.compile (kendo.core.js:439)
    at Object.r [as template] (jquery.js:10268)
    at kendo.groupable.js:109
    at Object.<anonymous> (kendo.groupable.js:91)
    at kendo.groupable.js:80
    at Object.1210 (kendo.groupable.js:79)
    at n (kendo.groupable.js:21)
    at Object.0 (kendo.groupable.js:49)
    at n (kendo.groupable.js:21)
    at e.exports.0 (kendo.groupable.js:41)
```

### Expected/desired behavior
no error should be throwed

### Environment

* **Kendo UI version:** 2019.1.320
* **jQuery version:** 3.3.1
* **Browser:** [Chrome 73.0] 

![image](https://user-images.githubusercontent.com/982868/55065092-ca977a80-50ad-11e9-8981-2844cf3fd6cc.png)

**Comment by zdravkov** [comment]
Hi @novalagung ,

I have tried to replicate the issue with a very simple Grid but to no avail. Here is my test [Stackblitz ](https://stackblitz.com/edit/jmt2hn?file=package.json)- would you please help me replicate the issue.

Greetings,
Plamen

**Comment by novalagung** [comment]
> Here is my test Stackblitz - would you please help me replicate the issue.

The issue only appear when the app consumes the distributable files (result of `yarn build` command). I don't get any issues on `yarn serve`.

Anyway I just found the root case and the solution. Turns out the issue happened because misconfigured CSP header. I need to add the `unsafe-eval` on csp `script-src`.

> before:
> `Content-Security-Policy: script-src 'self' 'unsafe-inline'`
>
> after:
> `Content-Security-Policy: script-src 'self' 'unsafe-inline' 'unsafe-eval'`

It was confusing me a lot since the error message is not very friendly (in my opinion).
